Makefile

```diff
@@ -1,9 +1,58 @@
 TOOL_VERSION = 2.1.12
 TEST_DATA = test/simpleTestForGenerator
-
+GIT_USERNAME=License Publisher (maintained by Gary O'Neall)
+GIT_EMAIL=gary@sourceauditor.com
+LICENSE_DATA_REPO=https://github.com/goneall/license-list-data.git
+LICENSE_DATA_REPO_NO_SCHEME=github.com/goneall/license-list-data.git
```

Comment:
To stay DRY, how about:

```
LICENSE_DATA_REPO_NO_SCHEME = github.com/goneall/license-list-data.git
LICENSE_DATA_REPO = https://$(LICENSE_DATA_REPO_NO_SCHEME)
```

Also, I think these variable definitions are more readable with whitespace around the `=` (the examples in [the docs][1] use whitespace).

[1]: https://www.gnu.org/software/make/manual/html_node/Setting.html